src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int maxNumber = 100;
+        int number = random.nextInt(maxNumber);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
**Think about which variables should be local in the method and which should be class-level**
Same as Random